### ISSUE TITLE
NPE in getQC

### ISSUE THREAD

**Issue by sjaenick** [MGX-metagenomics/MGX-gui#39]
In gitlab by @sjaenick on Mar 27, 2015, 10:02

empty seqrun results in

```
de.cebitec.mgx.client.exception.MGXServerException: <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><title>GlassFish Server Open Source Edition 3.1.2.2 - Error report</title><style type="text/css"><!--H1 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:22px;} H2 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} H3 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:14px;} BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;} B {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;} P {font-family:Tahoma,Arial,sans-serif;background:white;color:black;font-size:12px;}A {color : black;}HR {color : #525D76;}--></style> </head><body><h1>HTTP Status 500 - </h1><hr/><p><b>type</b> Exception report</p><p><b>message</b></p><p><b>description</b>The server encountered an internal error () that prevented it from fulfilling this request.</p><p><b>exception</b> <pre>javax.servlet.ServletException: java.lang.NullPointerException</pre></p><p><b>root cause</b> <pre>java.lang.NullPointerException</pre></p><p><b>note</b> <u>The full stack traces of the exception and its root causes are available in the GlassFish Server Open Source Edition 3.1.2.2 logs.</u></p><hr/><h3>GlassFish Server Open Source Edition 3.1.2.2</h3></body></html>
	at de.cebitec.mgx.client.access.rest.RESTMethods.catchException(RESTMethods.java:125)
	at de.cebitec.mgx.client.access.rest.RESTMethods.get(RESTMethods.java:72)
	at de.cebitec.mgx.client.access.rest.SeqRunAccess.getQC(SeqRunAccess.java:56)
	at de.cebitec.mgx.gui.controller.SeqRunAccess.getQC(SeqRunAccess.java:172)
```

**Comment by sjaenick** [comment]
In gitlab by @sjaenick on Mar 27, 2015, 15:29

```
[#|2015-03-27T15:26:05.034+0100|SEVERE|glassfish3.1.2|de.cebitec.mgx.web.SeqRunBean|_ThreadID=35;_ThreadName=Thread-2;|java.lang.NullPointerException
        at java.io.File.<init>(File.java:277)
        at de.cebitec.mgx.seqstorage.ReaderFactory.getReader(ReaderFactory.java:33)
        at de.cebitec.mgx.sequence.SeqReaderFactory.getReader(SeqReaderFactory.java:19)
        at de.cebitec.mgx.web.SeqRunBean.getQC(SeqRunBean.java:193)
```

**Comment by sjaenick** [comment]
In gitlab by @sjaenick on Mar 27, 2015, 17:04

mentioned in commit 60433bc76630d71e57c6f6adc9453ee41c5f03ba

**Comment by sjaenick** [comment]
In gitlab by @sjaenick on Mar 27, 2015, 17:04

Status changed to closed by commit 4d854567c8dafaf051e9f70eb3973182f6fb9fa2
